binaries/cli/src/template/c/dataflow-template.yml

```
@@ -1,24 +1,22 @@
 nodes:
-  - id: op_1
-    operator:
-      shared-library: build/op_1
+  - id: talker_1
+    custom:
+      source: bin/talker_1
       inputs:
-        foo: dora/timer/millis/100
+        tick: dora/timer/millis/100
       outputs:
-        - bar
-  - id: op_2
-    operator:
-      shared-library: build/op_2
+        - speech
+  - id: talker_2
+    custom:
+      source: bin/talker_2
       inputs:
-        foo: dora/timer/secs/2
+        tick: dora/timer/secs/2
       outputs:
-        - bar
+        - speech
 
-  - id: custom-node_1
+  - id: listener_1
     custom:
-      source: build/node_1
+      source: bin/listener_1
       inputs:
-        input-1: op_1/bar
-        input-2: op_2/bar
-      outputs:
-        - foo
+        input-1: talker_1/speech
```

I think that we should probably use speech so that we can showcase the de-serialization of the message
```suggestion
        speech: talker_1/speech
```